compiler/ballerina-treegen/src/main/java/io/ballerinalang/compiler/internal/treegen/targets/Target.java

```diff
@@ -55,16 +57,19 @@ public Target(TreeGenConfig config) {
         this.config = config;
     }
 
-    public abstract List<SourceText> execute(SyntaxTree syntaxTree);
+    public abstract List<SourceText> execute(SyntaxTree syntaxTree,
+                                             HashMap<String, TemplateNodeConfig> templateConfig);
```

Comment:
writing in-line would not make the check styles fail I guess

Reply:
yeah it's exactly 120 with the name refactoring, will fix.

Reply:
Changed back due to checkstyles failure